include/TrustWalletCore/TWTransactionHelper.h

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "TWBase.h"
+#include "TWCoinType.h"
+#include "TWData.h"
+#include "TWDataVector.h"
+#include "TWString.h"
+
+TW_EXTERN_C_BEGIN
+
+/// Non-core transaction utility methods, like building a transaction using an external signature.
+TW_EXPORT_CLASS
+struct TWTransactionHelper;
```

Comment:
How about `TWExternalTransaction` or other naming? to explicitly tell what does this `struct` do

Reply:
`TransactionHelper` is general, too general.
`TransactionBuilder` is also too general, and internally the term building is used in many chains already.
`TransactionCompiler` is my best alternative, I call compiling when putting together prepared transaction with signatures.

Reply:
Bitcoin has `PSBT` or we can use `Staged/Partial` transaction? compiler is slightly better

Reply:
Renamed to TransactionCompiler